.circleci/config.yml

```diff
@@ -133,7 +133,11 @@ references:
         echo "=== sourcing $BASH_ENV ==="
         source $BASH_ENV
         sudo mkdir -p -m 777 $CIRCLE_ARTIFACTS/
+
+        # Creating new clean logs folder
+        rm -rf $CIRCLE_ARTIFACTS/logs
```

Comment:
You were seeing logs from other jobs appearing in jobs where you `attach_workspace`?

Reply:
Previous jobs, not any other jobs, but yes.